### ISSUE TITLE
non-standard amino acids

### ISSUE THREAD

**Issue by hroetsc** [QuantSysBio/inSPIRE#16]
Hi,
I noticed that inSPIRE crashed when there is by accident a non-standard amino acid like X in a sequence. Could inSPIRE filter out these sequences and perhaps print a warning?
Thanks!

**Comment by jamc1996** [comment]
Hi Hanna,

Thanks for spotting this. The PR just merged should address the issue but is not yet available on pip. To install it please run the following steps.

Uninstall your existing installation:
```
pip uninstall inspirems
```

Clone the GitHub repo:
```
git clone https://github.com/QuantSysBio/inSPIRE.git
```

Change directory into inSPIRE:
```
cd inSPIRE
```

Install the cloned repo:
```
python setup.py develop
```

Thanks again and let me know if there are still any issues.

Best,
John.

**Comment by jamc1996** [comment]
Hi Hanna,

I'm closing this issue as the version containing the fix (inSPIRE 1.5) is now published on pip meaning it can now be installled via the standard:

```
pip install inspirems==1.5
```

This does not impact you if you successfully installed inSPIRE manually via the instructions above.

Thank you for your help impoving inSPIRE. We hope to hear from you again!

Best wishes,
John.
